kolibri/plugins/learn/assets/test/views/search-results-grid.spec.js

```diff
@@ -0,0 +1,126 @@
+import { mount, shallowMount } from '@vue/test-utils';
+import commonCoreStrings from 'kolibri.coreVue.mixins.commonCoreStrings';
+import { createTranslator } from 'kolibri.utils.i18n';
+import SearchResultsGrid from '../../src/views/LibraryPage/SearchResultsGrid.vue';
+/* eslint-disable import/named */
+import useSearch, { useSearchMock } from '../../src/composables/useSearch';
+/* eslint-enable import/named */
+
+const SearchStrings = createTranslator('SearchResultsGrid', SearchResultsGrid.$trs);
+const coreStrings = commonCoreStrings.methods.coreString;
+
+jest.mock('../../src/composables/useSearch');
+
+describe('when search results are loaded', () => {
+  /* useSearch#displayingSearchResults is truthy and isLoading is false */
+  it('does not display a ChannelCardGroupGrid', () => {
```

Comment:
I'd suggest renaming this in a way that you can understand what's not being displayed even without being familiar with `ChannelCardGroupGrid`, e.g. "does not display a list of channels".